azure-iot-device/tests/common/test_mqtt_transport.py

```diff
@@ -230,6 +230,31 @@ def test_configures_tls_context_with_server_verification_certs(self, mocker, moc
             cadata=server_verification_cert
         )
 
+    # TODO: Finalize these test args once the format for ciphers is determined
+    @pytest.mark.it(
+        "Configures TLS/SSL context with provided cipher(s) if present during instantiation"
+    )
+    @pytest.mark.parametrize(
```

Comment:
the parametrization here is an unnecessary complication of this test. If we're talking about not wanting an explosion of tests. We're just testing here that the input is plugged to the input to set_ciphers. The parametrization comes in test_pipeline_stages_mqtt

Reply:
The explosion I was referring to was at the client level, not here. However, I see the logic in removing this, as all validation happens by the time this is set in the config.

---
In reply to: [369286262](https://github.com/Azure/azure-iot-sdk-python/pull/434#discussion_r369286262) [](ancestors = 369286262)